docs/api/geometries/TubeBufferGeometry.html

```diff
@@ -35,11 +35,12 @@
 		<h2>Example</h2>
 
 		<code>
-		function CustomSinCurve( scale ) {
-
-			THREE.Curve.call( this );
```

Comment:
Why have you deleted this line? This breaks the inheritance of the curve object.